canonical-websites/maas.io
# Hero update

## Done

- Updates all row hero blocks to the new dark design with white text
- New content and styles for the home page row hero.
- Renames 'Get started' page to 'Install'

## QA

Pull down the brand and run ```npm i``` and ```make develop``` to view the development site. Check all new hero rows to be the new design and works fine on mobile.

## Issue / Card

_n/a_

## Screenshots

_n/a_
